centaur/src/main/resources/standardTestCases/string_interpolation_optional/string_interpolation_optional.wdl

```diff
@@ -0,0 +1,19 @@
+version development
+task string_interpolation_task {
+
+  input {
+    String? str
+  }
+
+  command <<<
+      echo ~{'"' + str + '"'}
+  >>>
+
+  output {
+    String out = read_string(stdout())
+  }
```

Comment:
It looks like a missing `docker` runtime attribute is responsible for the non-`Local` Centaur failures
```suggestion
  }
  
  runtime {
    docker: "ubuntu:latest"
  }
```